### ISSUE TITLE
Only getting Q1 and Q4 quarters

### ISSUE THREAD

**Issue by ZeningQu** [vega/vega-lite#1532]
![image](https://cloud.githubusercontent.com/assets/822034/18151326/49aaa462-6fa3-11e6-98be-10dc55097812.png)

```json
{
  "description": "Stock price mean per quarter broken down by years.",
  "data": {"url": "data/stocks.csv"},
  "mark": "point",
  "encoding": {
    "x": {"field": "date", "type": "temporal", "timeUnit": "quarter"},
    "y": {"field": "price", "type": "quantitative", "aggregate": "mean"},
    "color": {"field": "symbol", "type": "nominal"}
  }
}
```

Possibly related to https://github.com/vega/vega-lite/issues/1531

The data has all twelve months, but we are only getting Q1 and Q4

**Comment by willium** [comment]
If you remove the `*3` from https://github.com/vega/vega-lite/blob/master/src/datetime.ts#L160 this works as expected.

**Comment by kanitw** [comment]
Fixed in #1571 (pending review)
